core/src/autogluon/core/trainer/abstract_trainer.py

```diff
@@ -1209,6 +1217,12 @@ def _add_model(self, model: AbstractModel, stack_name: str = 'core', level: int
             del model
         return True
 
+    def _add_model_attr(self, model: str, **attributes):
```

Comment:
This function name gives people impression that it only add attributes. However, it actually update the attribute. Maybe rename to `_update_model_attr`?

Reply:
Good catch, updated!